test/parse.jl

```diff
@@ -662,3 +662,8 @@ end
 
 # issue #17701
 @test expand(:(i==3 && i+=1)) == Expr(:error, "invalid assignment location \"==(i,3)&&i\"")
+
+# PR #15592
+let str = "[1] [2]"
+    @test_throws ParseError parse(str)
```

Comment:
For future reference, it's fine to do `@test_throws ParseError parse("[1] [2]")`; you don't always need a variable.